snapshots/btrfs/btrfs.go

```diff
@@ -199,6 +237,9 @@ func (b *snapshotter) makeSnapshot(ctx context.Context, kind snapshots.Kind, key
 			return nil, err
 		}
 	}
+	if err = btrfs.QuotaCtl(target, true); err != nil {
```

Comment:
What is the cost of setting the quota in terms of extra memory, cpu, or disk?

Reply:
https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-quota#PERFORMANCE_IMPLICATIONS
says:
```
https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-quota#PERFORMANCE_IMPLICATIONS
```

Should we make this optional?
If so, what value should we return when quota is disabled?
We might be able to use "walking" method, but it's confusing to have two different definitions of disk usage. (So I suggest returning -1)